akka-testkit/src/main/resources/reference.conf

```diff
@@ -26,8 +26,18 @@ akka {
       type = akka.testkit.CallingThreadDispatcherConfigurator
     }
   }
-  
-  actor.serialization-bindings {
-    "akka.testkit.JavaSerializable" = java
+
+  actor {
+    serializers {
+      test-message-serializer = "akka.testkit.TestMessageSerializer"
+    }
+
+    serialization-identifiers {
+      "akka.testkit.TestMessageSerializer" = 23
+    }
+
+    serialization-bindings {
+      "akka.testkit.JavaSerializable" = java
```

Comment:
Then it'd be default automagic for everything java serializable so there would be no way of knowing if you accidentally used java serialization for some of your messages.